pkg/components/volume.go

```diff
@@ -22,24 +24,33 @@ func createVolumeClaims(specVolumeClaimTemplates []ytv1.EmbeddedPersistentVolume
 	return volumeClaims
 }
 
+func createConfigTemplateVolumeMount() v1.VolumeMount {
+	return v1.VolumeMount{
+		Name:      consts.ConfigTemplateVolumeName,
+		MountPath: consts.ConfigTemplateMountPoint,
+		ReadOnly:  true,
+	}
+}
+
 func createConfigVolumeMount() v1.VolumeMount {
 	return v1.VolumeMount{
 		Name:      consts.ConfigVolumeName,
 		MountPath: consts.ConfigMountPoint,
-		ReadOnly:  true,
+		ReadOnly:  false,
 	}
 }
 
 func createVolumeMounts(specVolumeMounts []v1.VolumeMount) []v1.VolumeMount {
 	volumeMounts := make([]v1.VolumeMount, 0, len(specVolumeMounts)+1)
 	volumeMounts = append(volumeMounts, specVolumeMounts...)
+	volumeMounts = append(volumeMounts, createConfigTemplateVolumeMount())
 	volumeMounts = append(volumeMounts, createConfigVolumeMount())
 	return volumeMounts
 }
 
-func createConfigVolume(configMapName string, mode *int32) v1.Volume {
+func createConfigTemplateVolume(volumeName string, configMapName string, mode *int32) v1.Volume {
```

Comment:
It's used for `consts.ConfigTemplateVolumeName` and for `consts.ConfigVolumeName`. So I think `createConfigVolume` is better name for this function.

Reply:
Fixed